bundle/Command/GenerateProjectCommand.php

```diff
@@ -190,4 +193,39 @@ protected function cleanup(): void
             // Do nothing
         }
     }
+
+    private function activateGitHooks(): void
+    {
+        if (!class_exists(Config::class)) {
+            return;
+        }
+
+        /** @var \Symfony\Component\Filesystem\Filesystem $fileSystem */
+        $fileSystem = $this->getContainer()->get('filesystem');
+        $projectDir = $this->getContainer()->getParameter('kernel.project_dir');
+        $captainHookFiles = [
+            $projectDir . '/captainhook_disabled.json',
+            $projectDir . '/captainhook_enabled.json',
+        ];
+
+        if (!$fileSystem->exists($captainHookFiles)) {
+            return;
+        }
+
+        if ($this->questionHelper->ask(
+            $this->input,
+            $this->output,
+            $this->getConfirmationQuestion(
+                    'Do you want to use git hooks suitable for project development?',
+                    false
+                )
+        )
+        ) {
+            $fileSystem->symlink($projectDir . '/captainhook_enabled.json', $projectDir . '/captainhook.json');
```

Comment:
This creates absolute symlinks. Should be created with relative paths:

In `netgen/site-bundle`, there's code similar to this, maybe it can help:

```
        $this->fileSystem->symlink(
            $this->fileSystem->makePathRelative(
                $source,
                realpath(dirname($destination))
            ),
            $destination
        );
```

Reply:
Fixed in fdc482e8d399c2513109b3f2d9108e76f87dfd2f